### DISCUSSION TITLE
My youtube channel

### DISCUSSION THREAD

**Discussion by stla** [pyvista/pyvista#2280] [Show and tell]
Hello,

  I started to do a [youtube channel](https://www.youtube.com/channel/UCQ2hC8ETS9FFm2pzzyZCVUw/videos) to store my animations. So far it contains only a few PyVista animations, but there will be more in the future. For each video, there is a link to the code in the description. Note that you can loop a video by right-clicking on it.

**Comment by banesullivan** [comment]
So cool!!! Thanks for posting all of those videos!

I'm subscribed
